kernel/boot/Handoff.cpp

```diff
@@ -156,15 +99,28 @@ void Handoff::parseStivale2(Handoff* that, void* handoff)
                 rs232_printf("\tResolution: %ix%ix%i\n",
                     framebuffer->framebuffer_width,
                     framebuffer->framebuffer_height,
-                    (framebuffer->framebuffer_bpp * 8));
+                    (framebuffer->framebuffer_bpp));
+                rs232_printf("\tPixel format:\n"
+                             "\t\tRed size:    %u\n"
+                             "\t\tRed shift:   %u\n"
+                             "\t\tGreen size:  %u\n"
+                             "\t\tGreen shift: %u\n"
+                             "\t\tBlue size:   %u\n"
+                             "\t\tBlue shift:  %u\n",
+                             framebuffer->red_mask_size,
+                             framebuffer->red_mask_shift,
+                             framebuffer->green_mask_size,
+                             framebuffer->green_mask_shift,
+                             framebuffer->blue_mask_size,
+                             framebuffer->blue_mask_shift);
                 // Initialize the framebuffer information
-                that->_fbInfo = FramebufferInfo(
+                that->_fbInfo = fb::FramebufferInfo(
                     framebuffer->framebuffer_width,
                     framebuffer->framebuffer_height,
                     framebuffer->framebuffer_bpp,
                     framebuffer->framebuffer_pitch,
                     (void*)framebuffer->framebuffer_addr,
-                    (FramebufferMemoryModel)framebuffer->memory_model,
+                    (fb::FramebufferMemoryModel)framebuffer->memory_model,
```

Comment:
Should this be a `static_cast`?

Reply:
Why would it need to be a static cast?

Reply:
Updating anyways because it's the C++ thing to do. We should open an issue for converting all C-style casts to C++ style casts.

Reply:
I'm not sure when which kind of cast is appropriate. I think static cast does some sanity checks at compile time to make sure the cast makes sense?